app/assets/stylesheets/views/_homepage_header.scss

```diff
@@ -0,0 +1,62 @@
+@import "govuk_publishing_components/individual_component_support";
+
+$pale-blue-colour: #d2e2f1;
+
+.app-c-homepage-header {
```

Comment:
Now that this has moved back to a view template, I think we should be able to remove the `app-c-` prefix from the class names.